src/lambda/commands/createNewSamApp.ts

```diff
@@ -352,21 +352,26 @@ async function validateSamCli(samCliValidator: SamCliValidator): Promise<void> {
 
 export async function getProjectUri(
     config: Pick<CreateNewSamAppWizardResponse, 'location' | 'name'>,
-    file: string
+    files: string[]
 ): Promise<vscode.Uri | undefined> {
-    const cfnTemplatePath = path.resolve(config.location.fsPath, config.name, file)
-    if (await fileExists(cfnTemplatePath)) {
-        return vscode.Uri.file(cfnTemplatePath)
-    } else {
-        vscode.window.showWarningMessage(
-            localize(
-                'AWS.samcli.initWizard.source.error.notFound',
-                'Project created successfully, but {0} file not found: {1}',
-                file,
-                cfnTemplatePath
-            )
-        )
+    let file: string = files[0]
+    let cfnTemplatePath: string = path.resolve(config.location.fsPath, config.name, file)
```

Comment:
can   `file` and `cfnTemplatePath` be uninitialized? Looks like they are reassigned in the loop below.
```suggestion
    let cfnTemplatePath: string
```

Reply:
That's what I had originally, but it was giving me an error further down that `file` and `cfnTemplatePath` were being used before they were assigned. Using the non-null assertion operator can overcome this.  Is this because the string array could be empty?

Reply:
> but it was giving me an error further down that `file` and `cfnTemplatePath` were being used before they were assigned

But assigning them to a value that is never used is not a good way to avoid such a warning... it just tricks the warning, and actually doesn't address the problem the warning detected.